### DISCUSSION TITLE
Deno support?

### DISCUSSION THREAD

**Discussion by d9k** [chakra-ui/panda#1518] [Ideas]
I don't see anything about Deno support in [documentation](https://panda-css.com/docs/installation/cli).

By the way, [Deno](https://deno.com) have excellent SSR frameworks: [ultra](https://ultrajs.dev), [aleph](https://alephjs.org), [fresh](https://fresh.deno.dev).

**Comment by astahmer** [comment]
don't take this as official doc/example but I had made a PoC with fresh a few months ago, just made it public
https://github.com/astahmer/panda-fresh
